conbench/tests/populate_local_conbench.py

```diff
@@ -522,13 +523,18 @@ def sample_slowdown(s):
         # a slowdown.
         if random.random() < 0.1:
             # Shift all samples a little higher (not by the same amount)
-            samples = [sample_slowdown(s) for s in samples]
+            samples = [sample_slowdown(s, offset=slowdown_offset) for s in samples]
+
+        # Simulate outliers with more significant slowdowns
+        if random.random() < 0.08:
+            # Shift all samples a little higher (not by the same amount)
+            samples = [sample_slowdown(s, offset=outlier_offset) for s in samples]
 
         # Simulate for any sample to have been affected by some blip-slowdown
         if random.random() < 0.02:
```

Comment:
note to self: the samples in here are the sub samples elsewhere 
This relates to https://github.com/conbench/conbench/pull/811#discussion_r1127723139